### PR TITLE
NetworkAdapter comments

### PR DESCRIPTION
**Pull request by HerbCaudill**
The "hmmm maybe not" there has made me question reality for long enough.

### CODE REVIEW

File: packages/automerge-repo/src/network/NetworkAdapter.ts [hunk 1]
```diff
@@ -11,27 +11,25 @@ import { StorageId } from "../storage/types.js"
  * until the adapter emits a `ready` event before it starts trying to use it
  */
 export abstract class NetworkAdapter extends EventEmitter<NetworkAdapterEvents> {
-  peerId?: PeerId // hmmm, maybe not
```
Comment (pvh) [inline packages/automerge-repo/src/network/NetworkAdapter.ts:14]: The "hmm, maybe not" here is that the network adapter gets its peerId at connect() time but we have it on the constructor. Perhaps we should clean that up?

Comment (HerbCaudill) [reply packages/automerge-repo/src/network/NetworkAdapter.ts:14]: ? It's not on the constructor - the abstract class doesn't specify a constructor interface, and none of the implementations take a peerId in the constructor, only in `connect()`.

Comment (pvh) [reply packages/automerge-repo/src/network/NetworkAdapter.ts:14]: yes -- of course, my misread there. thanks.

**Comment by pvh** [comment]
I manually reimplemented this due to the conflicts. At least I got rid of the "hmm".

**Pull request closed without merging**